resources/fetching/fetchers/gcp/monitoring_fetcher.go

```diff
@@ -57,23 +57,25 @@ var monitoringAssetTypes = map[string][]string{
 func (f *GcpMonitoringFetcher) Fetch(ctx context.Context, cMetadata fetching.CycleMetadata) error {
 	f.log.Info("Starting GcpMonitoringFetcher.Fetch")
 
-	monitoringAsset, err := f.provider.ListMonitoringAssets(monitoringAssetTypes)
+	monitoringAssets, err := f.provider.ListMonitoringAssets(monitoringAssetTypes)
```

Comment:
the main change in the monitoring fetcher is that it returns an array of assets instead of a single asset. this is because when scope is organization, we may have multiple projects, so now we get an array of assets where each asset is a different project. (note each asset consists of 2 asset types and a project id)